src/main/java/ai/lucidtech/las/sdk/CreatePredictionOptions.java

```diff
@@ -0,0 +1,45 @@
+package ai.lucidtech.las.sdk;
+
+import org.json.JSONObject;
+
+
+public class CreatePredictionOptions {
+    private int maxPages;
+    private Boolean autoRotate;
+
+
+    public CreatePredictionOptions(){
+        this.maxPages = 0;
```

Comment:
One way to be able to specify 'null' as default for int would be to use the Integer class. Setting maxPages to 0 might seem a bit strange when the semantics is more of the 'unspecified, use API default' kind.